.env/public

```diff
@@ -1,4 +0,0 @@
-export BLOCK_NUMBER='14633843'
```

Comment:
you could create a file `.env.sample` to list the environment variables the project expects. you can even add the ones that are not really private, something like:

```
BLOCK_NUMBER=14633843
DAI_WHALE=0xe78388b4ce79068e89bf8aa7f218ef6b9ab0e9d0
USDC_WHALE=0x47ac0fb4f2d84898e4d9e7b4dab3c24507a6d503
STKAAVE_WHALE=0x32b61bb22cbe4834bc3e73dce85280037d944a4d
ALCHEMY_KEY=
MNEMONIC=
```

Reply:
Why is this being removed?

Reply:
Right, just saw the `dotenvconfig` change. Massil's `.env.sample` is the way to go, you'll also need to update the docs to inform users they should copy that file over and fill in the missing values.